e2e/atlas/alerts_test.go

```diff
@@ -33,12 +33,89 @@ const (
 func TestAlerts(t *testing.T) {
 	atlasEntity := "atlas"
 	alertsEntity := "alerts"
-	alertID := "5ecbef6b2359825e889837a7"
+	var alertID string
 
 	cliPath, err := cli()
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
+	t.Run("List", func(t *testing.T) {
```

Comment:
the main think is to run listing first, grab the id from there instead of hardcoded and then use it for the rest